dev/sample_plugin_1_5/__init__.py

```diff
@@ -147,7 +147,7 @@ def update_item(self, item, caller=None, source=None, dest=None):
         """
         if caller != self.get_shortname():
             # code to execute, only if the item has not been changed by this this plugin:
-            logger.info("Update item: {}, item has been changed outside this plugin".format(item.id()))
+            logger.info("Plugin '{}': Update item: {}, item has been changed outside this plugin".format(self.get_fullname(), item))

```

Comment:
I guess `logger.info(...)` will fail here because it should be `self.logger.info(...)`.